doc/rst/lib_spi.rst

```diff
@@ -823,10 +939,360 @@ As application runs that reads a value from the SPI connected WiFi chip and prin
   5400
   Done.
 
+The value `5400` represents bits 15 to 0 of the default value of the CONFIG register of the WFM200.
+
+Likewise, the following two commands should yield the same console output:
+
+.. code-block:: console
+
+  xrun --xscope bin/SYNC/app_spi_master_SYNC_NO_CLKBLK.xe
+  xrun --xscope bin/SYNC/app_spi_master_ASYNC.xe
+
 
 |newpage|
 
 
+SPI Slave Example
+=================
+
+Introduction
+............
```

Comment:
A bit ocd, but for the master example "overview" is used, here its "introduction"

Reply:
Ah, because I had a clash when I put them one level up. I fixed that but forgot to revert. Will fix.